src/System Application/App/Upgrade Tags/src/UpgradeTags.Page.al

```diff
@@ -0,0 +1,32 @@
+// ------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+// ------------------------------------------------------------------------------------------------
+
+namespace System.Upgrade;
+page 9998 "Upgrade Tags"
```

Comment:
All other objects in this app have inherent permissions/entitlements. I assume this page needs that too?

Reply:
I will deal with that :-)